rollbar.env.sample

```diff
@@ -0,0 +1 @@
+ROLLBAR_API_KEY=<INSERT_KEY>
```

Comment:
do you need to add `rollbar.env` to `.gitignore`?

Reply:
no its already ignored, all *.env are ignored.